tests/integration/workflows/test_arxiv_workflow.py

```diff
@@ -233,3 +379,256 @@ def test_harvesting_arxiv_workflow_manual_accepted(
+            },
+            {
+                'cnum': 'C01-01-01',
+            }
+        ],
+    }
+
+    set_document_type_and_refereed(workflow, None)
+
+    assert 'refereed' not in workflow.data
+
+
+def test_workflows_set_refereed_true_if_one_journal_refereed(workflow_app, insert_journals_in_db, workflow):
+    workflow.data = {
+        '_collections': [
+            'Literature'
+        ],
+        'titles': [
+            'A title'
+        ],
+        'document_type': [
+            'conference paper',
+            'book',
+            'note',
+            'report'
+        ],
+        'publication_info': [
+            {
+                'journal_title': 'Test.Jou.1',
+                'journal_record':
+                    {
+                        '$ref': 'http://localhost:5000/api/journals/1936475'
+                    }
+            },
+            {
+                'cnum': 'C01-01-01',
+            },
+        ],
+    }
+
+    set_document_type_and_refereed(workflow, None)
+
+    assert workflow.data['refereed']
+
+
+def test_workflows_set_refereed_false_if_all_journals_not_refereed(workflow_app, insert_journals_in_db, workflow):
+    workflow.data = {
+        '_collections': [
+            'Literature'
+        ],
+        'titles': [
+            'A title'
+        ],
+        'document_type': [
+            'conference paper',
+            'book',
+            'note',
+            'report'
+        ],
+        'publication_info': [
+            {
+                'journal_title': 'Test.Jou.3',
+                'journal_record':
+                    {
+                        '$ref': 'http://localhost:5000/api/journals/1936477'
+                    }
+            },
+            {
+                'cnum': 'C01-01-01',
+            },
+            {
+                'journal_title': 'Test.Jou.4',
+                'journal_record':
+                    {
+                        '$ref': 'http://localhost:5000/api/journals/1936478'
+                    }
+            }
+        ],
+    }
+
+    set_document_type_and_refereed(workflow, None)
+
+    assert not workflow.data['refereed']
+
+
+def test_workflows_set_document_type_and_refereed_no_journal_records(workflow_app, insert_journals_in_db, workflow):
+    workflow.data = {
+        '_collections': [
+            'Literature'
+        ],
+        'titles': [
+            'A title'
+        ],
+        'document_type': [
+            'conference paper',
+            'book',
+            'note',
+            'report'
+        ],
+        'publication_info': [
+            {
+                'journal_title': 'Test.Jou.6'
+            },
+            {
+                'cnum': 'C01-01-01',
+            },
+            {
+                'journal_title': 'Test.Jou.4',
+                'journal_record':
+                    {
+                        '$ref': 'http://localhost:5000/api/journals/1936478'
+                    }
+            }
+        ],
+    }
+
```

Comment:
I would put this record, as well as the others in the tests above, in a fixture to make it more readable